web/ui/react-app/src/pages/Services.tsx

```diff
@@ -2,14 +2,113 @@ import React, { FC } from 'react';
 import { RouteComponentProps } from '@reach/router';
 import PathPrefixProps from '../PathPrefixProps';
 import { Alert } from 'reactstrap';
+import { useFetch } from '../utils/useFetch';
+import { LabelsTable } from './LabelsTable';
 
-const Services: FC<RouteComponentProps & PathPrefixProps> = ({ pathPrefix }) => (
-  <>
-    <h2>Service Discovery</h2>
-    <Alert color="warning">
-      This page is still under construction. Please try it in the <a href={`${pathPrefix}/service-discovery`}>Classic UI</a>.
-    </Alert>
-  </>
-);
+interface DiscoveredLabels {
```

Comment:
I think you can delete this interface (it contains some clearly testing field names, plus you never know what labels there will be anyway) and make any labels just the standard string->string map like in https://github.com/prometheus/prometheus/blob/213a8fe89a7308e73f22888a963cbf9375217cd6/web/ui/react-app/src/pages/targets/target.ts#L1-L3

Reply:
sorry. I had fixed this earlier, but the git revert would have created this again. Thanks for pointing.

Reply:
You can use ```Record<string, string>```